File: billing_back/billing_back/Models/Purchase.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace billing_back.Models
{
    public class Purchase 
    {
        public int IdPurchase { get; }
        public int IdBill { get; set; }
        public int IdProduct { get; set; }
        public int? ProductQuantity { get; set; }
        public float? PurchaseValue { get; set; }
    }
}

File: billing_front/src/app/shared/models/Purchase.ts
export interface Purchase {

  IdPurchase: number;
  IdBill: number;
  IdProduct: number;
  ProductQuantity: number;
  PurchaseValue: number;

}

export const PurchaseNull =
{
  IdPurchase: 0,
  IdBill: 0,
  IdProduct: 0,
  ProductQuantity: 0,
  PurchaseValue: 0
}
